src/cogent3/core/alignment.py

```diff
@@ -2486,9 +2486,18 @@ def probs_per_seq(self, motif_length=1, include_ambiguity=False, allow_gap=False
         )
         return counts.to_freq_array()
 
-    def entropy_per_seq(self, motif_length=1):
-        """returns shannon entropy per sequence"""
-        probs = self.probs_per_seq(motif_length=motif_length)
+    def entropy_per_seq(self,
+        motif_length=1,
+        include_ambiguity=False,
+        allow_gap=False,
+        exclude_unobserved=False,
+        alert=False):
+        """returns shannon entropy per sequence
```

Comment:
modify the docstring to match `_SequenceCollectionBase.counts_per_seq`, for the "notes" statement use restructured text, same as the "Parameters" section for `counts_per_seq`, but with "Notes"!
```
Notes
------

blah
```